### ISSUE TITLE
Role doesn't catch systems that mask systemd-timesyncd

### ISSUE THREAD

**Issue by kfiresmith** [idiv-biodiversity/ansible-role-systemd-timesyncd#3]
Discovered on fresh baseline builds of Ubuntu 20.04 that systemd-timesyncd is masked.  

```
TASK [idiv-biodiversity.systemd_timesyncd : enable systemd-timesyncd service and assure it is started] ******************************************************************************************************************************
fatal: [machine.college.edu]: FAILED! => {"changed": false, "msg": "Unable to enable service systemd-timesyncd: Failed to enable unit: Unit file /etc/systemd/system/systemd-timesyncd.service is masked.\n"}
```

Upon deeper inspection, it looks like Ubuntu uses a "fake" systemd-timesyncd package in the minimal server build and installs chrony alongside.  When you attempt to reinstall systemd-timesyncd it purges chrony and installs a real systemd-timesyncd package.  

I'm not sure the best way to work around this silly behavior.  Wanted to make sure it was on the board as a possible area to enhance the role.   I'll try to think about it some over coming days.

**Comment by wookietreiber** [comment]
@kfiresmith I've taken the time to add verification of the role via molecule, rewritten a major part of the tasks, and just released the changes as v1.3.0. The issue should be fixed, can you confirm this on your end?

**Comment by wookietreiber** [comment]
> @kfiresmith I've taken the time to add verification of the role via molecule, rewritten a major part of the tasks, and just released the changes as v1.3.0. The issue should be fixed, can you confirm this on your end?

Confirmed by https://github.com/idiv-biodiversity/ansible-role-systemd-timesyncd/pull/5#issuecomment-1702529516, closing.
